### PR TITLE
Some microoptimization

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Some small performance improvements. A few code style changes have been added by mistake (short array syntax).

### Manual testing scenarios (*)

n/a

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Review by fballiano — APPROVED** [review]
I think we had a PR to covert all arrays to short format but I guess this one shouldn't hurt anyway

File: app/code/core/Mage/Core/Model/Design/Package.php [hunk 1]
```diff
@@ -890,7 +890,7 @@ protected function _cssMergerImportCallback($match)
     protected function _cssMergerUrlCallback($match)
     {
         $quote = ($match[1][0] == "'" || $match[1][0] == '"') ? $match[1][0] : '';
-        $uri = ($quote == '') ? $match[1] : substr($match[1], 1, strlen($match[1]) - 2);
+        $uri = ($quote == '') ? $match[1] : substr($match[1], 1, -1);
```
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Design/Package.php:893]: Why `strlen($match[1])` is always 1?

File: app/code/core/Mage/Bundle/Model/Product/Type.php [hunk 2]
```diff
@@ -1009,8 +1009,8 @@ public function processBuyRequest($product, $buyRequest)
         $option     = $buyRequest->getBundleOption();
         $optionQty  = $buyRequest->getBundleOptionQty();
 
-        $option     = (is_array($option)) ? array_filter($option, 'intval') : array();
-        $optionQty  = (is_array($optionQty)) ? array_filter($optionQty, 'intval') : array();
+        $option     = (is_array($option)) ? array_filter($option, '\intval') : array();
```
Comment (kiatng) [inline app/code/core/Mage/Bundle/Model/Product/Type.php:1012]: Any reason you don't use short array here and other places?
```suggestion
        $option     = (is_array($option)) ? array_filter($option, '\intval') : []);
```

File: app/code/core/Mage/Core/Model/Design/Package.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Core/Model/Design/Package.php:893]: Its the same ....
```
$match[1] = 'abc';
var_dump(substr($match[1], 1, strlen($match[1]) - 2));
var_dump(substr($match[1], 1, -1));
```

File: app/code/core/Mage/Bundle/Model/Product/Type.php (hunk 2 above)
Comment (sreichel) [reply app/code/core/Mage/Bundle/Model/Product/Type.php:1012]: No reason - "A few code style changes have been added by mistake". Just kept it, b/c it doesnt hurt.

**Review by kiatng — APPROVED** [review]

File: app/code/core/Mage/Core/Model/Design/Package.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Core/Model/Design/Package.php:893]: @kiatng sorry, was to late - to try - to explain 😂

**Pull request merged**